views/requests/view.erb

```diff
@@ -1 +1,33 @@
-    Requests I've received
\ No newline at end of file
+
```

Comment:
give the page a <h2>Requests</h2> title in a .centered-container div to match the mock up

Reply:
added.